.idea/codeStyles/Project.xml

```diff
@@ -22,6 +22,7 @@
     <option name="DOWHILE_BRACE_FORCE" value="3" />
     <option name="WHILE_BRACE_FORCE" value="3" />
     <option name="FOR_BRACE_FORCE" value="3" />
+    <option name="SOFT_MARGINS" value="80" />
```

Comment:
I would discuss this with the core JxBrowser team as well.

Reply:
Discussed with @vladimir-ikryanov. 

Decided to set `80` for the hard wrapping, so that snippets were not long and fit well on the site.